app/controllers/movie_actors_controller.rb

```diff
@@ -0,0 +1,8 @@
+class MovieActorsController < ApplicationController
+  def create
+    movie = Movie.find(params[:id])
+    new_actor = Actor.find(params[:actor_id])
+    MovieActor.create!(movie: movie, actor: new_actor)
+    redirect_to "/movies/#{movie.id}"
+  end
+end
```

Comment:
You got it! The only thing I would mention is that this isn't necessarily a "new actor" so the naming could be confusing, but the implementation is correct. Great job.